docs/en/tutorials/training_detection_custom_dataset.md

```diff
@@ -0,0 +1,225 @@
+
+- `ValidatePolygons`: it filters out the bounding boxes that are outside of the image due to previous augmentations;
+
+- `ShrinkBinaryMap` and `BorderMap`: they make the binary map and the border map needed for dbnet training;
+
+- `NormalizeImage`: it normalizes the image by the mean and variance of the ImageNet dataset;
+
+- `ToCHWImage`: it changes `HWC` images to `CHW` images.
+
+For validation transform pipeline, all image augmentation operations are removed, and replaced by a simple resize function:
+
+```yaml
+eval:
+  dataset
+    transform_pipeline:
+      - DecodeImage:
+          img_mode: RGB
+          to_float32: False
+      - DetLabelEncode:
+      - DetResize:
+          target_size: [ 736, 1280 ]
+          keep_ratio: False
+          force_divisable: True
+      - NormalizeImage:
+          bgr_to_rgb: False
+          is_hwc: True
+          mean: imagenet
+          std: imagenet
+      - ToCHWImage:
+```
+More tutorials on transform functions can be found in the [transform tutorial](./transform_tutorial.md).
+
+### 2.3 Configure the model architecture
+
+Although different models have different architectures, MindOCR formulates them as a general three-stage architecture: `[backbone]->[neck]->[head]`. Take `configs/det/dbnet/dbnet_r50_icdar15.yaml` as an example:
+
+```yaml
+model:
+  type: det
+  transform: null
+  backbone:
+    name: det_resnet50  # Only ResNet50 is supported at the moment
+    pretrained: True    # Whether to use weights pretrained on ImageNet
+  neck:
+    name: DBFPN         # FPN part of the DBNet
+    out_channels: 256
+    bias: False
+    use_asf: False      # Adaptive Scale Fusion module from DBNet++ (use it for DBNet++ only)
+  head:
+    name: DBHead
+    k: 50               # amplifying factor for Differentiable Binarization
+    bias: False
+    adaptive: True      # True for training, False for inference
+```
+ The backbone, neck, and head modules are all defined under `mindocr/models/backbones`, `mindocr/models/necks`, and `mindocr/models/heads`.
+
+### 2.4 Configure Training hyperparameters
+
+Some training hyperparameters in `configs/det/dbnet/dbnet_r50_icdar15.yaml` are defined as follows:
+```yaml
+metric:
+  name: DetMetric
+  main_indicator: f-score
+
+loss:
+  name: L1BalancedCELoss
+  eps: 1.0e-6
+  l1_scale: 10
+  bce_scale: 5
+  bce_replace: bceloss
+
+scheduler:
+  scheduler: polynomial_decay
+  lr: 0.007
+  num_epochs: 1200
+  decay_rate: 0.9
+  warmup_epochs: 3
+
+optimizer:
+  opt: SGD
+  filter_bias_and_bn: false
+  momentum: 0.9
+  weight_decay: 1.0e-4
+```
+It uses `SGD` optimizer (in `mindocr/optim/optim.factory.py`) and `polynomial_decay` (in `mindocr/scheduler/scheduler_factory.py`) as the learning scheduler. The loss function is `L1BalancedCELoss` (in `mindocr/losses/det_loss.py`) and the evaluation metric is `DetMetric` ( in `mindocr/metrics/det_metrics.py`).
+
+
+## 3. Model Training, Evaluation, and Inference
```

Comment:
Please do not skip to dbnet training in 3. This is the critical part in this tutorial. Rec tutorial will also rewrite this part.

Reply:
OK，sure. I will revise it accordingly.